pkg/cvo/sync_worker_test.go

```diff
@@ -85,25 +85,21 @@ func Test_statusWrapper_ReportProgress(t *testing.T) {
 			w.Report(tt.next)
 			close(w.w.report)
 			if tt.want {
-				select {
```

Comment:
If there is single channel to read from then a `select-case` to read is not required.